client/lib/plans/constants.js

```diff
@@ -738,7 +840,96 @@ export const FEATURES_LIST = {
 		getDescription: () => i18n.translate(
 			'Custom polls, surveys, ratings, and quizzes for the ultimate in customer and reader engagement.'
 		)
-	}
+	},
+	[ FEATURE_CORE_JETPACK ]: {
+		getSlug: () => FEATURE_CORE_JETPACK,
+		getTitle: () => i18n.translate( 'Core Jetpack Services' ),
+		getDescription: () => i18n.translate(
+			'Stats, customization, and promotion tools'
+		),
+		hideInfoPopover: true
+	},
+	[ FEATURE_BASIC_SECURITY_JETPACK ]: {
+		getSlug: () => FEATURE_BASIC_SECURITY_JETPACK,
+		getTitle: () => i18n.translate( 'Basic Security' ),
+		getDescription: () => i18n.translate(
+			'Brute force protection, monitoring, secure logins, updates.'
+		),
+		hideInfoPopover: true
+	},
+	[ FEATURE_BASIC_SUPPORT_JETPACK ]: {
+		getSlug: () => FEATURE_BASIC_SUPPORT_JETPACK,
+		getTitle: () => i18n.translate( 'Basic Support' ),
+		getDescription: () => i18n.translate(
+			'Standard support for all Jetpack free features.'
+		),
+		hideInfoPopover: true
+	},
+	[ FEATURE_SITE_BACKUPS_JETPACK ]: {
+		getSlug: () => FEATURE_SITE_BACKUPS_JETPACK,
+		getTitle: () => i18n.translate( 'Site Backups' ),
+		getDescription: () => i18n.translate(
+			'Automated daily backups (unlimited storage), one-click restores, and 30-day archive.'
+		),
+		hideInfoPopover: true
+	},
+	[ FEATURE_REALTIME_BACKUPS_JETPACK ]: {
+		getSlug: () => FEATURE_REALTIME_BACKUPS_JETPACK,
+		getTitle: () => i18n.translate( 'Site Backups' ),
+		getDescription: () => i18n.translate(
+			'Automated real-time backups (unlimited storage), one-click restores, and unlimited archive.'
```

Comment:
Alternate string suggestion: 
* `translate( 'Real-time backups, unlimited archives, and one-click threat resolution. Also includes SEO tools, and unlimited video hosting.' )`&mdash; translations: **1**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/jetpack-plans-tab)